docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,284 @@
+Ignores
+========
+
+When creating Larva tests, you will find that some constantly changing values will make the tests fail at certain points because there is a difference between the current result of the test and the expected result of the test, for example, the “timestamp”. After all, time changes every second. For solving this kind of issue, Larva has several ways for ignoring parts of messages to compare them. This subsection is intended to name and describe all those ignores within the Larva testing tool.
+
+.. list-table:: Overview
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Options
+     - Description
+   * - ignoreRegularExpressionKey
+     - Replaces all occurrences of regular expression with
+      
+       IGNORE
+   * - ignoreKey 
+     - Replaces all matching occurrences of provided key with
+      
+       IGNORE  
+   * - ignoreContentBetweenKeys 
+     - Replaces all characters found between key1 and key2 with
+      
+       IGNORE
+   * - ignoreKeysAndContentBetweenKeys
+     - Replaces all characters found between key1 and key2, 
+       
+       including key1 and key2, with IGNORE
+   * - ignoreContentBeforeKey
+     - Ignores everything before provided key has been found
+   * - ignoreContentAfterKey
+     - Ignores everything after provided key has been found
+   * - ignoreCurrentTimeBetweenKeys       
+     - Ignore current time between key1 and key2
+   * - replaceRegularExpressionKeys
+     - Replaces all matches of regular expression (key1) with
+      
+       the value of key2
+   * - replaceKey
+     - Replaces all matches occurrences of provided key with
+      
+       provided value
+   * - replaceEverywhereKey     
+     - Replaces all matches occurrences of provided key with
+      
+       provided value, same as replaceKey
+   * - removeRegularExpressionKey
+     - Replaces all matches of regular expression with “”
+   * - removeKeysAndContentBetweenKeys
+     - Removes all characters found between key1 and key2,
+      
+       including key1 and key2
+   * - removeKey 
+     - Removes all matching occurrences of provided key
+   * - decodeUnzipContentBetweenKeys
+     - Decode and unzip content between key1 and key2
+   * - canonicaliseFilePathContentBetweenKeys
+     - Canonicalise filepath content between key1 and key2
+   * - formatDecimalContentBetweenKeys
+     - Format decimal content between key1 and key2
+
+
+Usually those ignores are written in the ``common.properties`` file of your Larva test, in this way they apply to all test scenarios which have included this ``common.properties`` file. If you want to use ignores in a particular test scenario, then you can put them in that related ``scenario.properties`` file.
```

Comment:
The reader may have forgotten about common.properties. In section "Defining your services" I introduced common.properties as the file where you define your services. Can you tell your readers that you can define ignores along with your services? And can you add a reference to the said section?